### ISSUE TITLE
Error: Execution of native.

### ISSUE THREAD

**Issue by nzkfc** [jay-fivem/qb-vape#1]
Been awhile since an update, is this a new bug or something we need to update/change in our qb-vape script?
https://i.imgur.com/FSdLdOR.png

**Comment by midz8** [comment]
Yeah when will this be fixed? As its spamming with the native error.

**Comment by nzkfc** [comment]
Any update?
